docs/release-notes/rbac-ntsc-mr.rst

```diff
@@ -0,0 +1,16 @@
+:orphan:
+
+**New Features**
+
+-  RBAC: Following on the initial RBAC support added in 0.19.7 the enterprise edition of Determined
+   (`HPE Machine Learning Development Environment
+   <https://www.hpe.com/us/en/solutions/artificial-intelligence/machine-learning-development-environment.html>`_)
+   has added support for Role-Based Access Control over new entities:
+
+   -  JupyterLab Notebooks, Tensorboards, Shells, and Commands are now housed under workspaces and
+      can have access to them restricted by role.
```

Comment:
nit: Maybe split into two sentences:
JupyterLab Notebooks, Tensorboards, Shells, and Commands are now housed under workspaces. Access to these tasks can now be restricted by role.

Reply:
I was trying to avoid calling them tasks but we do call them tasks in the webui so sounds good